go/client/cmd_log_send.go

```diff
@@ -226,19 +233,21 @@ func (c *CmdLogSend) GetUsage() libkb.Usage {
 }
 
 func (c *CmdLogSend) logFiles(status *fstatus) logs {
+	logDir := c.G().Env.GetLogDir()
 	if status != nil {
 		return logs{
 			desktop: status.Desktop.Log,
 			kbfs:    status.KBFS.Log,
 			service: status.Service.Log,
+			start:   filepath.Join(logDir, libkb.StartLogFileName),
```

Comment:
Yeah good to be consistent... fixed to make part of status.. I think I did it the right way?